include/FxLine.h

```diff
@@ -91,17 +95,22 @@ class FxLine : public QWidget
 	QColor m_strokeInnerInactive;
 	static QPixmap * s_sendBgArrow;
 	static QPixmap * s_receiveBgArrow;
-
-	QStaticText m_staticTextName;
+	bool m_inRename;
+	QString m_newName;
+	QLineEdit * m_renameLineEdit;
+	QGraphicsProxyWidget * m_proxyWidget;
+	QGraphicsView * m_view;
+	QGraphicsScene * m_scene;
+	QString m_name;
```

Comment:
This member is not needed. It is set to `Engine::fxMixer()->effectChannel( m_channelIndex )->m_name`.